src/apis/SteamStore.js

```diff
@@ -0,0 +1,32 @@
+const { APIWrapper } = require('../')
+const axios = require('axios')
+
+const API_URL = 'https://store.steampowered.com/api'
+
+module.exports = class SteamStore extends APIWrapper {
+  constructor () {
+    super({
+      name: 'steamstore'
+    })
+  }
+
+  search (query) {
+    return axios({
+      url: `${API_URL}/storesearch`,
+      params: {
+        term: query,
+        l: 'english',
+        cc: 'US'
```

Comment:
can't this be relative?

Reply:
it only accepts 2 char language codes, but I can try

Reply:
I could use the `l` param on the app info tho, the only issue is that it doesn't support values such as `pt-BR` or whatever, it only supports the full language name, like `english` or `portuguese`

Reply:
https://www.npmjs.com/package/iso-639 (this one is a bit heavy, there's more out there)
get the language code (`pt`), find on the table and get the name (`Portuguese`), lowercase it (`portuguese`), and use it on the `l`, and you can try to get the country (`BR`) to use on the `cc`. Some languages on the bot(like the furry one), just use english as default, and also remember to test the api to see if it works with unknown value (e.g. using random string on the `l` and `cc` to see if return error or just defaults to english) to handle future errors

Reply:
https://github.com/SwitchbladeBot/switchblade/pull/1242/commits/8f9a13de63af7e1e1f790586427536a5409ea338